src/Components/StockTrading.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/prop-types */
+import { Flex, Text } from "@sparrowengg/twigs-react";
+
+const StockTrading = ({ icon, title, borderBottom, bgColor }) => {
+  return (
+    <Flex
+      css={{
+        width: "250px",
+        paddingLeft: "10px",
+      }}
+      gap="32px"
+      alignItems="center"
+    >
+      <Flex
+        css={{
+          width: "78px",
+          height: "48px",
+          borderRadius: "15px",
+          backgroundColor: bgColor,
+        }}
+        justifyContent="center"
+        alignItems="center"
+      >
+        <img
+          src={icon}
+          alt=""
+          style={{
+            width: "20px",
+            height: "20px",
+          }}
+        />
+      </Flex>
+
+      <Flex
+        flexDirection="column"
+        css={{
+          height: "100%",
+          borderBottom: borderBottom ? "1px solid $secondaryLight" : "",
```

Comment:
@9xVibee instead of empty string can you use none

Reply:
it is giving error that none is not defined

Reply:
"none"